test/conformance/usm/urUSMSharedAlloc.cpp

```diff
@@ -0,0 +1,2 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: MIT
```

Comment:
This is missing files for testing `urMemFree` and `urMemGetMemAllocInfo` which are both int he [USM section](https://oneapi-src.github.io/unified-runtime/core/api.html#usm). There's perhaps some inconsistency with the naming of these entry points.

Reply:
Yes I was confused by this slightly - I suppose they should be called `urUSMMemFree` & `urUSMGetMemAllocInfo`?

Reply:
Something along those lines, although the `M` in `USM` stands for memory so not including `Mem` would remove some reduncandy form the names. An entry point name change is deffo out of scope for this PR though.

Reply:
https://github.com/oneapi-src/unified-runtime/issues/201